tests/unit/test_tools_archive.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Tests for webcompat.com archive tools.
+
+TODO: test that an issue can be on a different repo
+"""
+
+import datetime
+import json
+import logging
+import pathlib
+
+import pytest
+import requests
+from requests.exceptions import HTTPError
+
+from tools.archive import model
+
+
+def get_fixture(filename):
+    """Return the data fixture."""
+    current_root = pathlib.Path('.').resolve()
+    fixture_root = current_root / 'tests' / 'fixtures' / 'tools'
```

Comment:
This looks magical -- it implements the `/` operator to create (navigate) child paths.

https://docs.python.org/3/library/pathlib.html#operators

Reply:
@miketaylr 

* https://realpython.com/python-pathlib/
* https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/
* https://rednafi.github.io/digressions/python/2020/04/13/python-pathlib.html